test/test_lazy_clone.py

```diff
@@ -0,0 +1,216 @@
+# Owner(s): ["module: tests"]
```

Comment:
This is not the right owner haha
We should have a module: cow on gitub and put that as the owner